pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools<65.2', 'wheel']
+requires = ['setuptools', 'wheel']
```

Comment:
Older versions of setuptools break with Python 3.12, so it was necessary to remove this restriction.